xhprof_lib/config.sample.php

```diff
@@ -8,10 +8,11 @@
 $_xhprof['dbpass'] = 'password';
 $_xhprof['dbname'] = 'xhprof';
 $_xhprof['dbadapter'] = 'Pdo';
-$_xhprof['servername'] = 'myserver';
+$_xhprof['servername'] = 's01';
```

Comment:
This is because the table structure has a CHAR(3) in the current version, and when I first copied these settings, they ended up breaking all INSERTs because "myserver" is 8 chars long. I thought it would be more intuitive to have an example with 3 chars, so no need to change table structure, but it doesn't fail on the first try.